config/default.yaml

```diff
@@ -198,3 +198,5 @@ ecosystem:
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
     # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
     dockerRegistry: ""
+    # Extra origins allowed to do CORS to API
+    allowCors: []
```

Comment:
I think we should set the default value to be same as ecosystem.ui

Reply:
ecosystem.ui is already in by default. This for extra origins. I didn't want users to repeat the ui value here again.